ui/scss/util/_util.scss

```diff
@@ -397,3 +397,70 @@ a.no-decoration,
 .cb {
   clear: both;
 }
+
+@mixin cstudioLoadingBar($height, $color) {
+  .cstudio__loading-bar {
```

Comment:
The class inside of the mixin reduces it's reusability. Let the consumer set the class and the mixin simply adds the parameterized content